pandas/core/generic.py

```diff
@@ -5335,8 +5335,7 @@ def _is_mixed_type(self) -> bool_t:
 
     @property
     def _is_numeric_mixed_type(self) -> bool_t:
-        f = lambda: self._mgr.is_numeric_mixed_type
-        return self._protect_consolidate(f)
+        return self._mgr.is_numeric_mixed_type
```

Comment:
Since this property is only used in the function right below, I would just inline the `self._mgr.is_numeric_mixed_type` there